packages/cypress/src/generators/cypress-project/files/v9-and-under/src/plugins/index.js

```diff
@@ -11,7 +11,7 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-const { preprocessTypescript } = require('@nrwl/cypress/plugins/preprocessor');
+const { preprocessTypescript } = require('@nx/cypress/plugins/preprocessor');
```

Comment:
as mentioned above, not needed since we removed the preprocessor, but I can follow up a PR to remove it to not clutter up this PR.